chart/templates/deployment.yaml

```diff
@@ -35,6 +34,9 @@ spec:
               memory: "1.5Gi"
```

Comment:
Same above this line: you have an image: ... sdv hardcoded still.

Reply:
Yeah I think ti's okay. THat's just how the image is named.